pmoves/services/hi-rag-gateway-v2/app.py

```
@@ -914,6 +914,25 @@ async def _handler(msg):
 # Optional HRM decoder controller (lazily loads packs per-namespace)
 _hrm_controller = HrmDecoderController(get_decoder_pack)
 
+# Enhanced HRM dual-stream sidecar (P6 Phase 3)
+_enhanced_hrm = None
+try:
+    _hrm_pkg_dir = str(Path(__file__).resolve().parent / "hrm")
+    if _hrm_pkg_dir not in sys.path:
+        sys.path.insert(0, str(Path(__file__).resolve().parent))
+    from hrm import HRMSidecar, HRMSidecarConfig as _HRMCfg
+    _enhanced_hrm = HRMSidecar(_HRMCfg(
+        enabled=HRM_ENABLED,
```

Ruff (0.15.10)

[error] Undefined name `HRM_ENABLED`

(F821)